coral-component-table/src/scripts/TableRowAccessibilityState.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2019 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+/**
+ @class Coral.Table.Row.AccessibilityState
+ @classdesc The Table Row Accessibility State
+ @htmltag coral-table-row-accessibilitystate
+ @return {HTMLElement}
+ */
+export default () => document.createElement('coral-table-row-accessibilitystate');
```

Comment:
@majornista I don't see it in the examples markup so I wonder if we can't use a simple `div` here instead with a class `.coral-Table-row-accessibilityState` ? You could create a template file for it.